app/viewModels/PackageShippingMarksViewModel.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels
+
+import models.{CheckMode, Index, Mode, NormalMode}
+import play.api.i18n.Messages
+
+import javax.inject.Inject
+
+case class PackageShippingMarksViewModel(houseConsignmentIndex: Index, itemIndex: Index, mode: Mode) {
```

Comment:
Let's press pause on these view models until we've got a structure we're happy with. This looks to be copied from Jake's branch but there are changes pending.